index.js

```diff
@@ -914,7 +962,7 @@
   //.
   //. Takes a function and returns `this` if `this` is a Nothing; otherwise
   //. it returns a Just whose value is the result of applying the function to
-  //. this Just's value.
+  //. this Just's value. This method is compatible with [`S.map`](#map).
```

Comment:
This is a very nice addition. :)

Elsewhere in the docs we don't use the `S.` prefix when referring to other Sanctuary functions. Let's drop it here.